R/pkg_ref_cache_downloads.R

```diff
@@ -1,13 +1,12 @@
+#' @describeIn riskmetric_metadata_caching
 #' Cache a list of available help files as LaTeX objects
-#' @param n Number of days to look back with default value of 365 days
 #'
-#' @inheritParams pkg_ref_cache
 #' @family package reference cache
+#' @keywords internal
 #'
 #' @importFrom cranlogs cran_downloads
-#' @return a \code{pkg_ref} object
-#' @keywords internal
-#' @noRd
-pkg_ref_cache.downloads <- function(x, ..., n=365) {
-  cran_downloads(x$name, from=Sys.Date()-n, to=Sys.Date())
+#' @usage NULL
+#' @export
+pkg_ref_cache.downloads <- function(x, ..., n = 365) {
```

Comment:
The generic has a name argument:

```suggestion
pkg_ref_cache.downloads <- function(x, name, ..., n = 365) {
```

Not sure if this is why the help page shows up as:

> `pkg_ref_cache(downloads)`: Cache a list of available help files as LaTeX objects

It also happens with `pkg_ref_cache(source_control_url)`. 

On that same page (`?pkg_ref_cache`),  there are many lines repeated on the See Also: "Other package reference cache"